cdap-examples/pom.xml

```diff
@@ -43,6 +43,7 @@
     <module>UserProfiles</module>
     <module>WebAnalytics</module>
     <module>WordCount</module>
+    <module>WikipediaPipeline</module>
```

Comment:
Really? I don't remember that, but...I'd think alphabetical would be better...if you can't find that PR, but it seems like this is what was asked for, we can leave it...